app/views/appliance_calculator/index.html.haml

```diff
@@ -3,4 +3,8 @@
 .cads-prose
   %p Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
 
-  %p Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
\ No newline at end of file
+  %p Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+
```

Comment:
This is me being super duper nitpicky and I only noticed because I was enjoying that you're not just using the first bit of Lores Ipsum, but is there a reason for the two blank lines here? (Obviously not a blocker, just interested that linting doesn't shout)

Reply:
Oh interesting, I don't know why the linter hasn't complained!  It's temporary anyway.